docs/readthedocs/source/doc/Orca/Tutorial/yarn.md

```diff
@@ -330,11 +330,10 @@ ${SPARK_HOME}/bin/spark-submit \
     --num-executors 2 \
     --archives /path/to/environment.tar.gz#environment \
     --properties-file ${BIGDL_HOME}/conf/spark-bigdl.conf \
-    --py-files ${BIGDL_HOME}/python/bigdl-spark_${SPARK_VERSION}-${BIGDL_VERSION}-python-api.zip,model.py \
```

Comment:
do we need to modify this?

Reply:
It is ok to keep it, and in my tests it can be removed if user has installed bigdl package in archive.

Reply:
In this case, I just adjust the position of this term.

Reply:
okay.